### PR TITLE
Feature: Add/change Track for Event

### PR DESCRIPTION
**Pull request by hennevogel**
oSC16 is using special EventTypes to workaround this limitation. I don't see why this shouldn't be possible...

### CODE REVIEW

File: app/views/proposal/_tooltip.html.haml [hunk 1]
```diff
@@ -3,33 +3,21 @@
   %li{'class'=>class_for_todo(progress_status['registered'])}
     %span{'class'=>icon_for_todo(progress_status['registered'])}
       - if progress_status['registered']
```
Comment (differentreality) [inline app/views/proposal/_tooltip.html.haml]: Did you mean to remove the if statement too?

File: app/views/proposal/_tooltip.html.haml [hunk 2]
```diff
@@ -3,33 +3,21 @@
...
-        = link_to 'Edit your registration', edit_conference_conference_registrations_path(event.program.conference.short_title)
+        = link_to 'Register to the conference', edit_conference_conference_registrations_path(event.program.conference.short_title)
```
Comment (ChrisBr) [inline app/views/proposal/_tooltip.html.haml]: Why? If `if progress_status['registered']` you're already registered!

**Comment by ChrisBr** [comment]
Otherwise LGTM

File: app/views/proposal/_tooltip.html.haml (hunk 1 above)
Comment (hennevogel) [inline app/views/proposal/_tooltip.html.haml]: Indeed

Comment (hennevogel) [inline app/views/proposal/_tooltip.html.haml]: Oh no, the path is different. This is correct as it is

File: app/views/proposal/_tooltip.html.haml (hunk 2 above)
Comment (hennevogel) [inline app/views/proposal/_tooltip.html.haml]: The TODO list content should not change. The TODO is always 'Register to the conference' no matter if it's done or not.

**Comment by hennevogel** [comment]
So can we merge this?

**Comment by ChrisBr** [comment]
Yeah, let's :ship: it ...

**Pull request merged**